### PR TITLE
OCP4: Add rules to verify legal notice

### PR DESCRIPTION
**Pull request by JAORMX**
This is used to address NIST SP 800-53's AC-8 control, and will verify that a
legal notice has been set on the UI either a login template or a classification
banner. For the CLI, the legal notice will use the motd capabilities.

### CODE REVIEW

**Review by mrogers950 — APPROVED** [review]
Addressed my concerns in slack.
/lgtm

File: applications/openshift/general/openshift_motd_exists/rule.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  Display of a standardized and approved use notification before granting
+  access to the system ensures privacy and security notification
+  verbiage used is consistent with applicable federal laws, Executive Orders,
+  directives, policies, regulations, standards, and guidance.
+  <br /><br />
+  System use notifications are required only for access via login interfaces
+  with human users and are not required when such human interfaces do not
+  exist.
+
+identifiers: {}
```
Comment (jhrozek) [inline applications/openshift/general/openshift_motd_exists/rule.yml:61]: Please add a CCE

File: applications/openshift/general/oauth_login_template_set/rule.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  storing it in a Kubernetes Secret and referencing it in the appropriate
+  field as
+  {{{ weblink(link="https://docs.openshift.com/container-platform/4.7/web_console/customizing-the-web-console.html#customizing-the-login-page_customizing-web-console",
+              text="described in the documentation") }}}
+
+rationale: |-
+  Displays to users organization-defined system use notification message or banner before granting
+  access to the system that provides privacy and security notices consistent with applicable federal laws.
+
+identifiers: {}
```
Comment (jhrozek) [inline applications/openshift/general/oauth_login_template_set/rule.yml:19]: please add a CCE

**Review by jhrozek — CHANGES_REQUESTED** [review]
looks good except for the missing CCEs

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now f521ebe0b69c654ff2d4f45afc0f4575d0005bb5

**Review by jhrozek — APPROVED** [review]
lgtm

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 2c4890592f118f05d848367e207b78f86656c2ff

**Pull request merged**